core/src/toga/widgets/imageview.py

```diff
@@ -52,21 +41,3 @@ def image(self, image):
         if self._image is not None:
             self._impl.set_image(image)
             self.refresh()
```

Comment:
Setting `image = None` does not currently remove an existing image: this will have to be fixed.